test/expected/pg_dump.out

```diff
@@ -655,4 +655,10 @@ SELECT * FROM test_tz ORDER BY time;
 (4 rows)
 
 \c :TEST_DBNAME :ROLE_SUPERUSER
+-- make sure nobody is using it
+REVOKE CONNECT ON DATABASE :TEST_DBNAME_EXTRA FROM public;
+SELECT count(pg_terminate_backend(pg_stat_activity.pid)) AS TERMINATED
```

Comment:
I see two sources of flakiness here:
1) The query should return the same value regardless of the number of backends, for no backends as well, so something like `coalesce(0, 0 * sum(pg_terminate_backend(...)))`
2) If the process has already ended, it will output a WARNING, so we should add  `set client_min_messages = 'ERROR'`.

Reply:
Point 1 is probably addressed by `gset`, I was looking at an earlier version.

Reply:
@ante I disabled auto-merge, let's silence the warnings.

Reply:
There is a `\gset` at the end so it doesn't output anything but adding client messages to ERROR sounds like a good idea. Will add it.